### ISSUE TITLE
stuck on loading screen (on Ubuntu 22.04 using snap)

### ISSUE THREAD

**Issue by hikaen2** [jgraph/drawio-desktop#1197]
Hi,

I'm using drawio with snap on Ubuntu.
One day drawio started getting stuck on loading screen.

### Preflight Checklist

* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Describe the bug**

drawio stuck on loading screen.

**To Reproduce**

```
$ snap install drawio
$ drawio
```

**Expected behavior**

drawio not stuck on loading screen.

**Screenshots**
![image](https://user-images.githubusercontent.com/4222502/208809360-8f5dba8b-7881-4a67-9452-c9ba8215355a.png)

**draw.io version:**
- draw.io version 20.6.0

**Desktop:**
 - OS: Ubuntu 22.04 LTS
 - Browser: firefox
 -

**Comment by pnicorelli** [comment]
same here :)

![image](https://user-images.githubusercontent.com/2498926/208863249-25e41538-4fdf-4507-8849-8879587d91dd.png)

installed by snap on Ubuntu 22.04.1 LTS

the shell just says:

```
09:57:11.658 › SNAP env is defined, updater is disabled
WARNING: Kernel has no file descriptor comparison support: Operation not permitted
```

**Comment by cirrusj** [comment]
Running with `DRAWIO_ENV=dev drawio` shows the following error in the console:
```
Uncaught ReferenceError: icon is not defined
    at Menus.createMenubar (Menus.js:1308:8)
    at EditorUi.createUi (EditorUi.js:4517:62)
    at App.EditorUi [as constructor] (EditorUi.js:341:8)
    at new App (App.js:16:11)
    at realMain (App.js:919:48)
    at App.js:1008:5
    at mxClient.js:51:85
    at mxXmlRequest.<anonymous> (mxClient.js:139:183)
    at XMLHttpRequest.<anonymous> (mxClient.js:34:107)
```

**Comment by dyereh** [comment]
I have the same issue, but slightly different output in the console

```bash
/usr/share/libdrm/amdgpu.ids: No such file or directory
00:01:34.930 › SNAP env is defined, updater is disabled
```

I am also running ubuntu 22.04 & draw.io 20.6.0

**Comment by davidjgraph** [comment]
Fixed in 20.7.4, thanks for report.
